clients/unshaded/src/main/java/tachyon/client/ClientOptions.java

```diff
@@ -37,6 +37,8 @@
     private UnderStorageType mUnderStorageType;
     /** Worker location to execute this operation, if not possible, the operation will fail */
     private NetAddress mLocation;
+    /** TTL value passed by client */
```

Comment:
Could you clarify the comment a bit like `TTL for the file in Ms, after this much time has elapsed, the file will be automatically deleted`?